travis-build/base/Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG RUBY_VERSION=2.4
+FROM ruby:${RUBY_VERSION}
+
+LABEL authors="hoatle <hoatle@teracy.com>"
+
+# add more arguments from CI to the image so that `$ env` should reveal more info
+ARG CI_BUILD_ID
+ARG CI_BUILD_REF
+ARG CI_REGISTRY_IMAGE
+ARG CI_BUILD_TIME
+
+ENV CI_BUILD_ID=$CI_BUILD_ID CI_BUILD_REF=$CI_BUILD_REF CI_REGISTRY_IMAGE=$CI_REGISTRY_IMAGE \
+    CI_BUILD_TIME=$CI_BUILD_TIME
+
+RUN mkdir -p /opt/app
+
+ENV APP=/opt/app
+
+WORKDIR $APP
+
+RUN mkdir -p /builds
+
+RUN cd /builds; \
+    git clone https://github.com/travis-ci/travis-build.git; \
```

Comment:
should use the latest stable git tag version instead and make sure to clone with the latest change only (git clone --depth=1 for example) so that the docker image should be kept lightweight and minimal.

Reply:
make sure to have the git tag version as docker build argument so that we can have different tagged versions of travis-build.

For example:

teracy/travis-build:latest
teracy/travis-build:2.0.0

Reply:
fixed!